src/constants.js

```diff
@@ -0,0 +1,14 @@
+export const VERSION = '0.0.10'
+
+export const digidoc_mime = 'application/x-digidoc'
+export const digidoc_chrome = 'TokenSigning'
```

Comment:
These should not see light outside of their actual implementations. The idea of modules is to keep things modular and self-contained. In fact, proliferation of "digidoc" throughout the codebase should be avoided or kept minimal.